Platform/AlderlakeBoardPkg/Library/FspmUpdUpdateLib/FspmUpdUpdateLib.c

```diff
@@ -689,4 +689,13 @@ UpdateFspConfig (
       }
     }
   }
+#if FixedPcdGetBool(PcdFusaSupport) == 1
```

Comment:
"== 1" could be removed since PcdFusaSupport is boolean

Reply:
Fixed